rerun_py/tests/unit/test_space_view_blueprint.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import annotations
+
+import itertools
+
+from rerun.blueprint.archetypes.space_view_blueprint import SpaceViewBlueprint
+from rerun.blueprint.components.entities_determined_by_user import (
+    EntitiesDeterminedByUser,
+    EntitiesDeterminedByUserBatch,
+)
+from rerun.blueprint.components.included_query import IncludedQuery, IncludedQueryBatch
+from rerun.blueprint.components.space_view_class import SpaceViewClass, SpaceViewClassBatch
+from rerun.blueprint.components.space_view_origin import SpaceViewOrigin, SpaceViewOriginBatch
+from rerun.blueprint.components.visible import Visible, VisibleBatch
+from rerun.components.name import Name, NameBatch
+
+from .common_arrays import none_empty_or_value, uuid_bytes0, uuid_bytes1
+
+
+def test_space_view_blueprint() -> None:
+    class_identifier_arrays = ["3D", SpaceViewClass("3D")]
+    display_name_arrays = ["3D View", Name("3D View"), None]
+    space_origin_arrays = ["/robot/arm", None, SpaceViewOrigin("/robot/arm")]
+    entities_determined_by_user_arrays = [False, EntitiesDeterminedByUser(False), None]
+    contents_arrays = [[uuid_bytes0, uuid_bytes1], [IncludedQuery(uuid_bytes0), IncludedQuery(uuid_bytes1)], None]
+    visible_arrays = [False, Visible(False), None]
+
+    all_arrays = itertools.zip_longest(
```

Comment:
FYI, zip_longest appears to lead to mypy lint errors.  In my PR I had to add:
```
        container_kind = cast(ContainerKindLike, container_kind)
        display_name = cast(Optional[Utf8Like], display_name)
        contents = cast(Optional[EntityPathArrayLike], contents)
        col_shares = cast(Optional[ColumnShareArrayLike], col_shares)
        row_shares = cast(Optional[RowShareArrayLike], row_shares)
        active_tab = cast(Optional[EntityPathLike], active_tab)
        visible = cast(Optional[VisibleLike], visible)
        grid_columns = cast(Optional[GridColumnsLike], grid_columns)
```
to fix it :sob: